app/src/main/java/dev/arkbuilders/arkmemo/data/repositories/GraphicNotesRepo.kt

```diff
@@ -0,0 +1,108 @@
+package dev.arkbuilders.arkmemo.data.repositories
+
+import android.util.Log
+import dev.arkbuilders.arklib.computeId
+import dev.arkbuilders.arklib.user.properties.PropertiesStorageRepo
+import dev.arkbuilders.arkmemo.data.ResourceMeta
+import dev.arkbuilders.arkmemo.di.IO_DISPATCHER
+import dev.arkbuilders.arkmemo.di.PropertiesStorageModule.STORAGE_SCOPE
+import dev.arkbuilders.arkmemo.models.Content
+import dev.arkbuilders.arkmemo.models.GraphicNote
+import dev.arkbuilders.arkmemo.preferences.MemoPreferences
+import dev.arkbuilders.arkmemo.utils.SVG
+import kotlinx.coroutines.CoroutineDispatcher
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.withContext
+import java.nio.file.Files
+import javax.inject.Inject
+import javax.inject.Named
+import kotlin.io.path.extension
+import kotlin.io.path.fileSize
+import kotlin.io.path.getLastModifiedTime
+import kotlin.io.path.name
+import kotlin.io.path.createTempFile
+
+class GraphicNotesRepo @Inject constructor(
+    private val memoPreferences: MemoPreferences,
+    @Named(IO_DISPATCHER) private val iODispatcher: CoroutineDispatcher,
+    @Named(STORAGE_SCOPE) private val storageScope: CoroutineScope,
```

Comment:
Is there any reason that you need to use @Named annotation? Can we just inject them as normal object?

Reply:
```CoroutineDispatcher``` and ```CoroutineScope``` have different objects that  we can provide using DI. For example, right now we need ```Dispatchers.IO``` and later we may need ```Dispatchers.Main```, how do we differentiate these if both are exposed as ```CoroutineDispatcher``` type?

Reply:
@ShubertMunthali `storageScope` and `propertiesStorageRepo` are not being used anywhere at the moment.
Any special reason we still want to keep them?

Reply:
Yes, we need to clean up